inc/views/base/announcements/announcements.twig

```diff
@@ -0,0 +1,30 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.forum_announcement }}</title>
+{% endblock head %}
+
+{% block body %}
+
+<table border="0" cellspacing="{{theme.borderwidth}}" cellpadding="{{theme.tablespace}}" class="tborder tfixed clear no_bottom_border">
```

Comment:
Very small nitpick, but pesonally I like to have a space between the `{{` and the variable name 😄